TraxxasParameters:
  diff_gear_ratio: 2.85 #ratio from spur gear rotation to wheel rotation
  wheel_radius: 0.05 #meters
  max_right_steering_angle: -0.77 # these default values tend to give 1 degree accuracy #-0.5235987756 #radians (-30 deg)
  max_left_steering_angle: 0.54 #0.5235987756 #radians (30 deg)
  steering_angle_is_in_radians: true #false means angle is in degrees
  max_speed: 10.0
  test_speed: 1.0

#  max_forward_vel: 10.0 #meters/sec (just an estimated guess for allowable speed)
#  min_forward_vel: 0.0 #find out later
#  max_reverse_vel: -10.0 #meters/sec (just an estimated guess for allowable speed)
#  min_reverse_vel: -0.0 #find out later, (don't get confused here; numerically, min_reverse_vel > max_reverse_vel)

  max_right_servo_feedback_val: 694 #as read from the servo when steering is at full 30 degrees right
  max_left_servo_feedback_val: 309 #as read from the servo when steering is at full 30 degrees left
  max_capable_speed: 13.0

#  max_capable_forward_speed: 13.0 #(I monitored between 12 and 13 m/s) #17.8816 #40 mph assumed
#  max_capable_reverse_speed: 13.0 #17.8816 #might be less than forward vel

  #parameters not loaded by the traxxas driver node
  max_accel: 0.05 #2.0 m/s^2, (!must be set correctly to prevent the vehicle from popping a wheely). A value of 0.05 works well to avoid the ESC break issue.

  esc_time_const: 0.3; #This is the constant Tp for the ESCs PID controller. (a smaller number gives a more overdamped response, this is most desirable while testing the vehicle, and preventing oscillations in velocity control)
